main/main.cpp

```diff
@@ -3614,6 +3637,10 @@ int Main::start() {
 				export_patch = true;
 			} else if (E->get() == "--patches") {
 				patches = E->next()->get().split(",", false);
+			} else if (E->get() == "--generate-pot") {
+				ERR_FAIL_COND_V_MSG(!editor && !found_project, EXIT_FAILURE, "Please provide a valid project path when generating POT file, aborting.");
+				editor = true;
```

Comment:
I think this and the above condition are wrong since `editor` is already set to true in the initial `--generator-pot` handling. I copied it from another option before tweaking the option handling later. I think it should just be `ERR_FAIL_COND_V_MSG(!found_project, EXIT_FAILURE, "Please provide a valid project path when generating POT file, aborting.");`. I'm going to take another look at this.

Reply:
Looking again, this does appear to be correct considering the way that `found_project` and `editor` are handled in `Main::setup`. Since it matches the way the export options are handled, I'm going to leave it as I had it.